static/js/library/import/directory.js

```diff
@@ -78,7 +91,7 @@ function connect(event, elem){
     var last_response_len = false;
     $.ajax(url_base + "/ajax/scan_library_directory", {
         method: "POST",
-        data: {"directory": directory, "minsize": minsize, "recursive": recursive},
+        data: {"directory": directory, "minsize": minsize, "recursive": recursive, "skipduplicatedirs": skipduplicatedirs, "maxresults": maxresults},
```

Comment:
Codacy found an issue: ['directory' is not defined.](https://app.codacy.com/manual/saucebarbeque/Watcher3/pullRequest?prid=6020720)